clients/venice-client/src/main/java/com/linkedin/venice/fastclient/RetriableAvroGenericStoreClient.java

```diff
@@ -296,33 +297,41 @@ public void streamingBatchGet(
     };
 
     TimeoutProcessor.TimeoutFuture scheduledRetryTask =
-        timeoutProcessor.schedule(retryTask, longTailRetryThresholdForBatchGetInMicroSeconds, TimeUnit.MICROSECONDS);
+        timeoutProcessor.schedule(retryTask, longTailRetryThresholdInMicroSeconds, TimeUnit.MICROSECONDS);
```

Comment:
From the code, batchget/compute doesn't support error retry.
Is this true?
If it is true, we need to add error retry since error can happen because of metadata out of sync and we don't have to update this PR for this since the PR is already too large.

Reply:
Yes. I think the test cases also mention that error retry is only supported for single get. I'm not sure why that's the case, though